pkg/endpoint/lock.go

```diff
@@ -66,5 +66,5 @@ func (e *Endpoint) unconditionalRLock() {
 
 // logDisconnectedMutexAction gets the logger and logs given error with context
 func (e *Endpoint) logDisconnectedMutexAction(err error, context string) {
-	e.getLogger().WithError(err).Error(context)
+	e.getLogger().WithError(err).Debug(context)
```

Comment:
Given that this is only ever called after failing `rlockAlive()` and the only error condition therein is `ErrNotAlive = errors.New("rlock failed: endpoint is in the process of being removed")`, this is the right thing to do.